.isort.cfg

```diff
@@ -0,0 +1,7 @@
+[settings]
```

Comment:
Also put this into `setup.cfg` in a `[isort]` section, to reduce the number of configuration files

Reply:
done